package.py

```diff
@@ -303,6 +378,41 @@ def setup_miniupnpc(self) -> None:
         zip_path.unlink(missing_ok=True)
         shutil.rmtree(extraction_dir)
 
+    @log_group('certificates')
+    def import_signing_certificates(self) -> bool:
```

Comment:
you are returning a boolean but not using it.

Also in some cases you `exit(1)` directly.

Try to do either or and use the return value at the caller if you have it.

Also specify in the docstring how to use this function

Reply:
At the moment it is unused because I am missing the standalone backend signing work. Like we do for macOS.

True means that the certificate is properly setup and a signed binary is going to be produced. 

It will be used in the PR that will add the sign method. 

Exit happens in a fatal case where you have configured a certificate but the key is missing.